src/library/dlgtagfetcher.cpp

```diff
@@ -222,14 +315,67 @@ void DlgTagFetcher::apply() {
             QDateTime::currentDateTimeUtc());
 }
 
+void DlgTagFetcher::retry() {
+    results->clear();
+    disconnect(m_track.get(),
+            &Track::changed,
+            this,
+            &DlgTagFetcher::slotTrackChanged);
+
+    addDivider(tr("Original tags"), results);
```

Comment:
Can we get around this?

Reply:
If it is okay, I will mark these two conversation as resolved since we continue them on #4871 .

Reply:
Yes, dies it make sence to rebase this on #4871?

Reply:
Yeah, that would be nice. should I wait for it to be in final situation?